Orchestration/OrchestrationService/ConfigurationBuilder.ps1

```diff
@@ -65,10 +65,56 @@ Class ConfigurationBuilder {
             $this.ReplaceTokens($this.configurationInstance);
 
         # Convert to hashtable and return Configuration Instance
-        return `
+        $configurationInstanceAsHashtable = `
             ConvertTo-HashTable `
                 -InputObject $this.configurationInstance;
+        $configurationInstanceAsObject = `
```

Comment:
any reason on using a different variable instead of $this.configurationInstance?